addon/controllers/building/edit/entrance/edit/link-street.js

```diff
@@ -6,20 +6,26 @@ import { task, lastValue } from "ember-concurrency-decorators";
 import { languageOptions } from "ember-ebau-gwr/models/options";
 
 export default class BuildingEditEntranceLinkStreetController extends Controller {
-  queryParams = ["locality"];
+  queryParams = ["locality", "PLZ4", "PLZ6"];
 
   @service street;
   @service buildingEntrance;
   @service building;
   @service notification;
   @service intl;
 
+  // A locality is uniquely described by either
+  // locality name + PLZ4 or PLZ4 + PLZ6
   @tracked locality = null;
+  @tracked PLZ4 = null;
+  @tracked PLZ6 = null;
```

Comment:
Why `= null`?

Reply:
Uhh I was just following the ember documentation on query parameters. There they initialize them to `null`.